tests/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import find_packages, setup
+
+setup(name="kafkatest",
+      version="0.8.3-SNAPSHOT",
```

Comment:
Agree with @ewencp - Apache is pretty strict about separating work-in-progress artifacts and public-consumable artifacts, and they have a bunch of legal reasons for that: http://www.apache.org/dev/release.html#releases

An alternative to grabbing version out of build.gradle is a small release script that will update the version everywhere at once.